.vscode/tasks.json

```diff
@@ -14,11 +14,18 @@
       "detail": "tree-sitter parse ${file}",
       "problemMatcher": []
     },
+    {
+      "label": "ts-gen-and-parse",
+      "type": "shell",
+      "command": "F=${file}; bin/generate-parser && { npx tree-sitter parse $F>\"${F%.*}.exp\"; code -r \"${F%.*}.exp\"; }",
+      "detail": "npx tree-sitter generate && tree-sitter parse ${file}",
+      "problemMatcher": []
+    },
```

Comment:
What do you think of just updating the `ts-parse` task with this?

Reply:
Sure 🤷